test/performance-tests.sh

```diff
@@ -61,23 +35,10 @@ function perf_tests() {
   local ip=$(kubectl get svc knative-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[*].ip}")
   local host=$(kubectl get route observed-concurrency -o jsonpath="{.status.domain}")
   
-  wait_until_routable "$ip" "$host"
+  wait_until_routable "$ip" "$host" || fail_test "App is not routable"
```

Comment:
s/App/Cluster/?

Reply:
Also, for consistency and better failure handling, don't fail here but return the proper error code.